sdk/src/main/java/com/google/cloud/dataflow/sdk/util/BigQueryTableRowIterator.java

```diff
@@ -423,8 +451,8 @@ private TableReference executeQueryAndWaitForCompletion()
   // client - BQ request to be executed
   // error - Formatted message to log if when a request fails. Takes exception message as a
   // formatter parameter.
-  public static <T> T executeWithBackOff(AbstractGoogleClientRequest<T> client, String error,
-      Object... errorArgs) throws IOException, InterruptedException {
+  public static <T> T executeWithBackOff(AbstractGoogleClientRequest<T> client, String error)
```

Comment:
this looks backwards-incompatible. Add a handler for the old method that indirects through `String.format`?